packages/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/NodesManager.java

```diff
@@ -471,15 +474,16 @@ public String obtainProp(int viewTag, String propName) {
           return "Unable to resolve background color";
         }
 
-        String invertedColor = String.format("%08x", (0xFFFFFFFF & actualColor));
+        String invertedColor = String.format("%08x", (actualColor));
```

Comment:
Do we really need `(` and `)` here?
```suggestion
        String invertedColor = String.format("%08x", actualColor);
```